.travis.yml

```diff
@@ -1,4 +1,5 @@
-# A Travis CI configuration file.
```

Comment:
might as well just use the long form.

Reply:
scancode.py does not accept the full license header in these files yet?

Reply:
My bad - this was a user error. I've now fixed this as instructed.